packages/test-utils/sdks/petstore/src/index.ts

```diff
@@ -3,6 +3,18 @@ import type { ConfigOptions, FetchResponse } from '@readme/api-core/types';
 import APICore from '@readme/api-core';
 import definition from '@readme/oas-examples/3.0/json/petstore.json';
 
+/**
+ * Swagger Petstore
+ *
+ * This is a sample server Petstore server.  You can find out more about Swagger at
+ * [http://swagger.io](http://swagger.io) or on [irc.freenode.net,
+ * #swagger](http://swagger.io/irc/).  For this sample, you can use the api key
+ * `special-key` to test the authorization filters.
+ *
+ * @see {@link http://swagger.io/terms/ Terms of Service}
+ * @license Apache-2.0
+ * @see {@link http://www.apache.org/licenses/LICENSE-2.0.html}
```

Comment:
Figured it'd be nice to add this link to the license if they have `info.license.url` but I'm indifferent on it. Might also be nice that if their `info.license.name` and `info.license.identifier` aren't recognized but they have a URL that we could still at least add that license URL in here?

Reply:
ah good point, maybe we can just flag that as a warning then? maybe our general spec validation process should check for valid SPDX identifiers too

Reply:
works for me! this might be a good idea for the publishing flow, i'll look into it. i'd argue that `@readme/openapi-parser` should eventually throw errors for invalid SPDX identifiers but not a priority by any means

Reply:
yeah i'm just worried about folks using a license like "proprietary" which isn't recognized by the SPDX so we'd have to add in exclusions. feels like a losing battle but i also don't know what all different licenses we've got for specs in our DB

Reply:
yeah my rationale here is that [the spec](https://spec.openapis.org/oas/v3.1.0#license-object) says the `info.license.identifier` should be an SPDX expression, and if it's not SPDX then you should be using the `info.license.url` field. but yeah fair point re: the handling of existing specs — cool with whatever you decide here!